shell/run_shell_cmd.go

```diff
@@ -57,10 +68,13 @@ func RunShellCommand(terragruntOptions *options.TerragruntOptions, command strin
 
 // Run the specified shell command with the specified arguments. Capture the command's stdout and return it as a
 // string.
-func RunShellCommandAndCaptureOutput(terragruntOptions *options.TerragruntOptions, command string, args ...string) (string, error) {
+func runShellCommandAndCaptureOutput(terragruntOptions *options.TerragruntOptions, copyWorkingDir bool, command string, args ...string) (string, error) {
 	stdout := new(bytes.Buffer)
 
 	terragruntOptionsCopy := terragruntOptions.Clone(terragruntOptions.TerragruntConfigPath)
+	if copyWorkingDir {
+		terragruntOptionsCopy.WorkingDir = terragruntOptions.WorkingDir
+	}
 	terragruntOptionsCopy.Writer = stdout
 	terragruntOptionsCopy.ErrWriter = stdout
 
```

Comment:
@brikis98 I noticed that the following "unmodified" code has the effect of ignoring the output in case of error, which is bad. I get error on Windows with the init and the get command in the case where I have modules in my module. I think it is a terragrunt bug on Windows that I will check later. But the effect was that I didn't get the error using terragrunt and I though that it was caused by my modifications, but it appears that the problem was there before.